packages/manager/apps/dedicated/client/app/account/user/user.controller.js

```diff
@@ -44,4 +55,12 @@ export default class UserAccountCtrl {
       }
     }.bind(this);
   }
+
+  descriptionOfHeading() {
+    const getRegion = this.coreConfig.getRegion();
```

Comment:
Regarding repetitive ask for `getRegion`, consider to create a `currentRegion` variable into constructor.